src/react_native/core.cljs

```diff
@@ -134,26 +134,46 @@
 
 (def use-context react/useContext)
 
+(defn use-ref-atom
+  [value]
+  (let [ref (use-ref (atom value))]
+    (.-current ^js ref)))
+
+(defn get-js-deps [deps]
+  (let [prev-deps (use-ref-atom {:prev-val false
```

Comment:
would need a better name i guess like `trigger-change`

Reply:
which one?

Reply:
`prev-val`

Reply:
mhm it's not actually trigger-change , because it will change from true to false, so at some point false will trigger change, so i will rename it to value

Reply:
`prev-val`